### DISCUSSION TITLE
v3.3.3

### DISCUSSION THREAD

**Discussion by crim50n** [minios-linux/minios-live#27] [Announcements]
## v3.3.3
- In Standard, Maximum and Ultra pulseaudio-module-bluetooth has been added.
- The kernel has been updated to 6.1.67.
- Updated all packages as of the current date.
- Mini Commander was added to initrd, you can start it in debug mode with mc command.

**Full Changelog**: https://github.com/minios-linux/minios-live/compare/v3.3.2...v3.3.3

<hr /><em>This discussion was created from the release <a href='https://github.com/minios-linux/minios-live/releases/tag/v3.3.3'>v3.3.3</a>.</em>

**Comment by PinguimPratico** [comment]
Where can I locate the hash of the ISO file to verify that it hasn’t been corrupted?

**Reply by crim50n** [reply]
Nowhere, unfortunately. In future release we will add sum files for checking.

**Comment by samopedia** [comment]
I can not find any hash files in /download/v4.1.2/ for minios-bookworm-xfce-standard-amd64-4.1.2.iso either.

I have generated sha256 hash file for my downloaded file but I have no way of checking if my download is OK.

3a793b600bf1e988bc05ef2fc7d588d63367c9a464e8e0289e7a2259c329183f    *minios-bookworm-xfce-standard-amd64-4.1.2.iso
